requirements.txt

```diff
@@ -11,6 +11,7 @@ pyasn1==0.1.7
 pytest==2.3.5
 pytest-mozwebqa
 pytest-xdist==1.8
+pytz===2014.10
```

Comment:
I'll stick to not pinning -- I prefer this too but had seen a potential convention formed on this project.
And oops on the fat-finger with the extra `=`